src/magento1/harness.yml

```diff
@@ -13,6 +13,14 @@ harness('inviqa/magento1'):
 ---
 attributes:
   app:
+    owner: www-data
+    group: www-data
+    dirs_and_files:
+      - '/app/public/app/etc/local.xml'
```

Comment:
Bit strange to have this writeable by magento - https://github.com/continuouspipe/dockerfiles/blob/1e63f8532ec7a6adab4d474987f70540fd2a1271/magento1/usr/local/share/magento1/magento_functions.sh#L16 isn't doing that so perhaps remove this line.
```suggestion
```